app/src/main/java/org/apache/taverna/mobile/ui/adapter/EndlessRecyclerOnScrollListener.java

```diff
@@ -28,7 +28,7 @@
     private int previousTotal = 0; // The total number of items in the dataset after the last load
     private boolean loading = true; // True if we are still waiting for the last set of data to
     // load.
-    private int visibleThreshold = 1; // The minimum amount of items to have below your current
+    private int visibleThreshold = 5; // The minimum amount of items to have below your current
```

Comment:
initialize visibleThreshold = 1 because we don't want to API Request till that user will reach to the last element